tests/scripts/analyze_outcomes.py

```diff
@@ -63,13 +72,25 @@ def _has_word_re(words: typing.Iterable[str]) -> typing.Pattern:
 
     IGNORED_TESTS = {
         'test_suite_psa_crypto_generate_key.generated': [
-            PSA_MECHANISM_NOT_IMPLEMENTED_SEARCH_RE,
+            # Ignore mechanisms that are not implemented, except
+            # for public keys for which we always test that
+            # psa_generate_key() returns PSA_ERROR_INVALID_ARGUMENT
+            # regardless of whether the specific key type is supported.
+            _has_word_re((mech
+                          for mech in _PSA_MECHANISMS_NOT_IMPLEMENTED
+                          if not mech.startswith('ECC_PUB')),
+                         exclude=r'ECC_PUB'),
         ],
         'test_suite_psa_crypto_not_supported.generated': [
             PSA_MECHANISM_NOT_IMPLEMENTED_SEARCH_RE,
         ],
         'test_suite_psa_crypto_op_fail.generated': [
-            PSA_MECHANISM_NOT_IMPLEMENTED_SEARCH_RE,
+            # Ignore mechanisms that are not implemented, except
+            # for test cases that assume the mechanism is not supported.
+            _has_word_re(_PSA_MECHANISMS_NOT_IMPLEMENTED,
+                         exclude=(r'.*: !(?:' +
```

Comment:
Question: would the effect be the same with
```
                         exclude=(r'.*Config: !(?:' +
```
or we need to match other cases as well?

Reply:
This is in `test_suite_psa_crypto_op_fail.generated`, not in `test_suite_config`. A matching test case description looks like
```
PSA mac CBC_MAC: !CBC_MAC with CAMELLIA
```